portfolio/src/main/webapp/script.js

```diff
@@ -27,8 +27,22 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-function addGreeting(){
-    fetch('/data').then(response => response.text()).then((quote) => {
-      document.getElementById('quote-container').innerText = quote;
+function addMessage() {
+  fetch('/data').then(response => response.json()).then((message) => {
+    const messageList = document.getElementById('message-container');
+    messageList.innerHTML ='';
+    messageList.appendChild(
```

Comment:
Would it be possible to write a loop that goes over the messages and automatically create a list element?

The idea would be that if we add more messages, we wouldn't need to change the code

Reply:
With the current json object, it might be difficult to implement a loop because the json isn't an array (it's an object with 3 fields, named `message1`, `message2` and `message3`) 

If you use Gson, it would instead convert your list into a Json array, and I believe you'd then be able to loop doing:

```javascript
for (var i = 0; i < message.length; i++) {
  messageList.appendChild(message[i]);
}
```

Reply:
Okay, thank you for the clarification.